media/css/firefox/private_browsing/private-browsing.less

```diff
@@ -546,3 +546,8 @@ html[lang^='en'] {
         width: @widthMobile;
     }
 }
+
+html, body{
+    max-width: 100%;
+    overflow-x: hidden;   
+}
```

Comment:
This looks like it fixes the issue. Nice!

For organizational consistency, can you move this change to the top of the file? It should be the first rule after the `// Common`` comment.

Thanks!